packages/plugin-ext/src/api/plugin-api.ts

```diff
@@ -68,6 +68,11 @@ export interface Plugin {
     lifecycle: PluginLifecycle;
 }
 
+export interface ConfigStorage {
```

Comment:
I do not insist, but please consider another name, like `PluginDirectories`

Reply:
Done. Take a look, please.